src/resources/HostedInterfacesCore/HostedInterfaceCoreConfig.model.ts

```diff
@@ -126,6 +126,13 @@ export interface ListHostedInterfacesParams extends Paginated {
     perPage?: number;
 }
 
+export interface ListHostedInterfacesByProjectParams extends ListHostedInterfacesParams {
+    /**
+     * The project for which the interfaces should be fetched
+     */
+    projectId?: string;
```

Comment:
I wonder if we should not have a generic project interface, that all the resources who have to deal with projects could extend

Reply:
Normally i would agree, but in this case, it is just a single parameter. It feels a bit overkill to me. If you say it is necessary though, I will do it !

Reply:
It would make more sense for the result templates in my mind though. I will add an interface for that at least.

Reply:
fair enough !